doc/members/accept_recovery.rst

```diff
@@ -21,17 +21,26 @@ A member proposes to recover the network and other members can vote on the propo
         }
     }
 
-    $ ./scurl.sh https://<ccf-node-address>/gov/propose --cacert network_cert --key member1_privk --cert member1_cert --data-binary @accept_recovery.json -H "content-type: application/json"
+    $ ./scurl.sh https://<ccf-node-address>/gov/proposal --cacert network_cert --key member1_privk --cert member1_cert --data-binary @accept_recovery.json -H "content-type: application/json"
     {
-        "completed": false,
-        "id": 1
+        "proposal_id": 1,
+        "proposer_id": 0,
+        "state": "OPEN"
     }
 
-    $ ./scurl.sh https://<ccf-node-address>/gov/vote --cacert network_cert --key member2_privk --cert member2_cert --data-binary @vote_accept_1.json -H "content-type: application/json"
-    false
+    $ ./scurl.sh https://<ccf-node-address>/gov/proposal/1/vote --cacert network_cert --key member2_privk --cert member2_cert --data-binary @vote_accept.json -H "content-type: application/json"
+    {
+        "proposal_id": 1,
+        "proposer_id": 0,
+        "state": "OPEN"
+    }
 
-    $ ./scurl.sh https://<ccf-node-address>/gov/vote --cacert network_cert --key member3_privk --cert member3_cert --data-binary @vote_accept_1.json -H "content-type: application/json"
-    true
+    $ ./scurl.sh https://<ccf-node-address>/gov/proposal/1/vote --cacert network_cert --key member3_privk --cert member3_cert --data-binary @vote_accept.json -H "content-type: application/json"
+    {
+        "proposal_id": 1,
```

Comment:
Why return anything other than the state here?

Reply:
We want to make it clear that the response is about the proposal, not the vote. So `"ACCEPTED"` is ambiguous, though perhaps `{ "proposal_state": "ACCEPTED" }` is fine. Its maybe useful to return consistent metadata about the proposal? But maybe not - the full state can be retrieved from `GET /proposals/{id}`, and the proposal_id itself is already obvious from the URL path.

Reply:
I agree `proposal_state` is an improvement, I suspect we can remove the rest.

Alternatively, we could take a minimalist view and return application/text "PROPOSAL_PASSED".

Reply:
I think we should stick to JSON objects for now. Once we have general OpenAPI support for other return formats, we can do a pass and go minimal, but for now I think the generality of JSON objects makes the API easier to use. On that line, I don't think its worth trimming fields out at the moment, if they're doing no harm?